bank/sql_bank/README.md

```diff
@@ -33,5 +22,5 @@ Run against an existing cockroach node or cluster.
 # Launch your node or cluster in secure mode with certificates in [mycertsdir]
 # Find a reachable address:[mycockroach:26257].
 # Run the example with:
-./sql_bank https://mycockroach:26257/?certs=mycertsdir
+./sql_bank "postgres://root@mycockroach:26257?sslcert=mycertsdir/root.client.crt&sslkey=mycertsdir/root.client.key"
```

Comment:
Ah. Nit: use single quotes so it's clear there's no expansion going on.

On Fri, Feb 26, 2016 at 11:58 AM, David Taylor <notifications@github.com>
wrote:

> In bank/sql_bank/README.md
> <https://github.com/cockroachdb/examples-go/pull/39#discussion_r54269677>:
>
> > @@ -33,5 +22,5 @@ Run against an existing cockroach node or cluster.
> >  # Launch your node or cluster in secure mode with certificates in [mycertsdir]
> >  # Find a reachable address:[mycockroach:26257].
> >  # Run the example with:
> > -./sql_bank https://mycockroach:26257/?certs=mycertsdir
> > +./sql_bank "postgres://root@mycockroach:26257?sslcert=mycertsdir/root.client.crt&sslkey=mycertsdir/root.client.key"
>
> & in the URL.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/examples-go/pull/39/files#r54269677>.
>